scripts/build_feature_store.py

```diff
@@ -953,6 +955,8 @@ def transform_building_features() -> DataFrame:
 # COMMAND ----------
 
 # DBTITLE 1,Apply upgrade functions
+# TODO: put this in some kind of shared config that can be used across srcipts/repos
+SUPPORTED_UPGRADES = [0.0, 1.0, 3.0, 4.0, 6.0, 9.0, 11.05, 13.01]
```

Comment:
Do we perhaps want to have individual constants for these numbers. I guess or people who work with these every single day, they know what 3 and 7 mean, but for the rest of us, constants would be amazing.

Reply:
@nweires where are the enums that you and @christine-george  are gonna use gonna live? Can import thsoe use those here?

Reply:
I'm still working out exactly what this will look like, but yes, there should be something better to use soon! I'm hoping to move away from using these upgrade ID numbers as the key to identify upgrades.

Draft changes are [here](https://github.com/rewiringamerica/pep/compare/natalie/split_upgrade_project_tables?expand=1#diff-87d50c70f25becb2cd023db89941a9083fff4fd24dcf6584e1e1f506df088de2) - see upgrades.json and the schema in read_upgrades.py.

Reply:
thanks Natalie! I will integrate these enums in a subsequent PR when that is ready to roll!